From 5234b48eb0765b7ed3e7836213747fbb3b0ee2b5 Mon Sep 17 00:00:00 2001
From: Rico Tzschichholz <ricotz@ubuntu.com>
Date: Sun, 4 Nov 2018 19:36:08 +0100
Subject: [PATCH] renderer: Fix type-argument mismatch

GLib.List<Rygel.DLNAProfile> vs GLib.List<weak Rygel.DLNAProfile>
---
 src/librygel-renderer/rygel-media-renderer-plugin.vala | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/librygel-renderer/rygel-media-renderer-plugin.vala b/src/librygel-renderer/rygel-media-renderer-plugin.vala
index 2d6cef1..cfb7b23 100644
--- a/src/librygel-renderer/rygel-media-renderer-plugin.vala
+++ b/src/librygel-renderer/rygel-media-renderer-plugin.vala
@@ -42,7 +42,7 @@ public class Rygel.MediaRendererPlugin : Rygel.Plugin {
 
     /// A list of DLNA profiles the MediaRenderer in this plug-in will accept
     /// for rendering.
-    public unowned GLib.List<unowned DLNAProfile> supported_profiles {
+    public unowned GLib.List<DLNAProfile> supported_profiles {
         get {
             return _supported_profiles;
         }
-- 
1.8.3-rc3

